src/Microsoft.Experimental.Collections/Microsoft.Experimental.Collections.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <Description>This package provides collections that are being considered for future versions of .NET Core.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageTags>collections, system, microsoft, fastdictionary, multivaluedictionary</PackageTags>
+    <VersionPrefix>1.0.4</VersionPrefix>
```

Comment:
Why are we adding a custom version prefix?

We have one coming from common.props:
https://github.com/dotnet/corefxlab/blob/3e2932bdb580538be39cccb9cc1fa306bb41cc02/tools/common.props#L6

Reply:
This package already exists in NuGet

https://www.nuget.org/packages/Microsoft.Experimental.Collections/1.0.3-alpha

So, unfortunately we need to have a custom version for this package unaligned for the other packages in this repo.